### PR TITLE
fix: render explicit string literals for representing boolean values

### LINKED ISSUE #2784: spectra information panel not rendering the boolean object correctly in the table
Screenshot 2023-12-03 at 19 40 37

### PR DESCRIPTION
**Pull request by hamed-musallam**

### CODE REVIEW

**Review by stropitek — APPROVED** [review]

File: src/components/info-panel/InfoPanel.tsx [hunk 1]
```diff
@@ -133,7 +133,7 @@ export function InfoPanel(props: InfoPanelProps) {
 function valueCell(value: number | string | object | boolean) {
   switch (typeof value) {
     case 'boolean':
-      return <ValueRenderers.Boolean value={value} />;
+      return <ValueRenderers.Boolean renderValue={String} value={value} />;
```
Comment (lpatiny) [inline src/components/info-panel/InfoPanel.tsx:136]: Why not just calling a Text renderer with String(value) ? I don't see why we should change the Boolean renderer.

Comment (hamed-musallam) [reply src/components/info-panel/InfoPanel.tsx:136]: we did this to render the value as a string "true" or "false" instead of a symbol

Comment (targos) [reply src/components/info-panel/InfoPanel.tsx:136]: @lpatiny's suggestion would give the same result.

Comment (hamed-musallam) [reply src/components/info-panel/InfoPanel.tsx:136]: The purpose is to provide users of the ValueRenderers.Boolean component with the flexibility to customize the rendering of strings, allowing them to choose representations that best convey true or false values according to their preferences

**Comment by stropitek** [comment]
We discussed this and agreed that showing `Yes` / `No` as values would be a better default than `true` / `false`

**Pull request merged**